### PR TITLE
FISH-1498: Client Certificate Validation documentation

### PR DESCRIPTION
**Pull request by rdebusscher**

### CODE REVIEW

File: docs/modules/ROOT/pages/documentation/payara-server/server-configuration/security/certificate-realm-certificate-validation.adoc [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+When you return `true` as the method result, the processing of the request continues. In the case false is returned, a `LoginException` is thrown resulting in a status 401 for the request.
+
+This class is loaded through the Java ServiceLoader mechanism. Make sure you have the following file. _META-INF/services/fish.payara.security.client.ClientCertificateValidator_ containing the fully qualified name of your implementation.
+
+----
+com.company.certificate.MyCertificateValidator
+----
+
+NOTE: Since the class is loaded through the Java ServiceLoader mechanism, you cannot object CDI or EJB beans into it. You can retrieve an instance of a CDI bean using  `CDI.current().select(MyCDIBean.class).get()`.  When the tenant Control code available in Payara Community is stabilised, it will be possible to directly inject those beans.
```
Comment (lprimak) [inline docs/modules/ROOT/pages/documentation/payara-server/server-configuration/security/certificate-realm-certificate-validation.adoc:52]: object -> inject

**Review by lprimak — CHANGES_REQUESTED** [review]
one slight word change and one-character syntax fix

File: docs/modules/ROOT/pages/documentation/payara-server/server-configuration/security/certificate-realm-certificate-validation.adoc [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+To have the `ClientCertificateValidator` interface available in your application, add the `payara-api` artifact to your application with the provided scope.
+
+
+[source, xml]
+----
+        <dependency>
+            <groupId>fish.payara.api</groupId>
+            <artifactId>payara-api</artifactId>
+            <version>{currentVersion}/version>
```
Comment (lprimak) [inline docs/modules/ROOT/pages/documentation/payara-server/server-configuration/security/certificate-realm-certificate-validation.adoc:23]: missing '<' in front of `</version>`

**Comment by rdebusscher** [comment]
Thx @lprimak! Updated according your remarks.

**Comment by AlanRoth** [comment]
I left some requests for changes on the Community version of this documentation, if they are implemented can they be reflected in the Enterprise documentation also to keep everything consistent

**Review by lprimak — APPROVED** [review]

**Comment by rdebusscher** [comment]
@AlanRoth Updated according your comments on community documentation PR

**Review by AlanRoth — APPROVED** [review]

**Pull request merged**